park-dashboard/src/Pages/NationalParkCharts/ActivitiesVsNP.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import axios from "axios";
+import { Doughnut } from "react-chartjs-2";
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from "chart.js";
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+const useStyles = makeStyles((theme) => ({
+  padding: {
+    paddingTop: "70px",
+  },
+  chartContainer: {
+    position: "relative",
+    height: "100vh",
+    width: "100vh",
+  },
+}));
+
+function ActivitiesVsNP() {
+  const [activitiesObject, setActivitiesObject] = useState({});
+  const classes = useStyles();
+  const fetchActivities = async () => {
+    axios
+      .get(
+        `https://developer.nps.gov/api/v1/activities/parks?API_Key=OaR7jmqSa22JAcsym9lVfStp58LmCqH9JdZUPEH7`
+      )
+      .then((res) => {
+        const activities = res.data.data;
+
+        let activitiesObject = {};
+
+        activities.forEach((activity) => {
+          const parks = activity.parks.filter(
+            (park) => park.designation === "National Park"
+          );
+          if (parks.length < 15) return;
+          activitiesObject[activity.name] = parks;
+        });
+        console.log(activitiesObject);
+        setActivitiesObject(activitiesObject);
+      });
+  };
+
+  useEffect(() => {
+    fetchActivities();
+  }, []);
+
+  let data = {
+    labels: Object.keys(activitiesObject),
+    datasets: [
+      {
+        label: "Number of National Parks",
+        data: Object.values(activitiesObject).map((parks) => {
+          return parks.length;
+        }),
+        backgroundColor: [
```

Comment:
Add more colors so colors can't be repeated.

Reply:
I have hard coded unique colors for each activity.

<img width="1507" alt="image" src="https://github.com/rutujapadgilwar/Park-Dashboard/assets/114516950/e1b29b06-a34d-4f99-b719-8bb141b6235d">